content/en/docs/best-practices/channel-naming.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Channel Naming
+linkTitle: Channel Naming
+description: This guide shows OLM users how to use and choose naming conventions for channels to manage their operator upgrades. 
+---
+
+## CHANNELS
+
+Operator Lifecycle Manager (OLM) provides a channel concept that allows you 
+as the operator author a means to specify a set of update streams for your 
+operator.
+
+Operator authors deal with two key tasks associated with OLM channels, first, 
+how to define a channel for your operator and then lastly how to interact 
+with OLM to deploy your operator using channels.   
+
+For each version of an operator you can specify a channel that it will belong 
+to. Since there can be multiple versions of an operator within a channel, 
+there is a notion of the latest version within a channel, or the channel head 
+version.  It's the channel head that OLM will install for most subscriptions.  
+
+There can also be multiple channels for a given operator package which is 
+used to offer different support models (e.g. pre-release, production).  Here 
+is a diagram that shows the relationship of operator versions to channels:
+
+![Channel Naming Image](/docs/conventions/best-practices/channel-naming1.png)
```

Comment:
Looks like this image is not showing up properly in the preview? Is the link correct?